app/src/main/java/com/kickstarter/libs/qualifiers/HorizontalRewards.kt

```diff
@@ -0,0 +1,6 @@
+package com.kickstarter.libs.qualifiers
+
+import javax.inject.Qualifier
+
+@Qualifier
+annotation class HorizontalRewards
```

Comment:
Let's rename this to `HorizontalRewardsPreference`

Reply:
Fixed!